sources/packages/api/src/route-controllers/assessment/models/assessment.dto.ts

```diff
@@ -1,17 +1,23 @@
 import { StudentAssessmentStatus } from "../../../services/student-assessment/student-assessment.models";
 import {
+  ApplicationExceptionStatus,
   Assessment,
   AssessmentTriggerType,
   OfferingIntensity,
   StudentAppealStatus,
 } from "../../../database/entities";
 import { ApiProperty } from "@nestjs/swagger";
 
+export enum RequestAssessmentTypeAPIOutDTO {
```

Comment:
This enum was created at the DTO level because it makes sense only at the DTO level. The name is proper, I believe. What is the issue with the enum name? The fact that it includes the `APIOutDTO` on it?

Reply:
mm, i was thinking if the enum is a DTO or not? IMO, it's more like a type of a DTO. For example, if we have a DTO named `AOutDTO`. The type-specific to the DTO will be something like `AOutDTOType`. Just my thought, let me know @andrewsignori-aot , @dheepak-aot  @guru-aot  @henryoforeh-aot  thoughts

Reply:
Are you mentioning it only for an enum? We have other specific types that are used only for DTOs and we never changed the naming convention, right? What would be the benefit here?

Reply:
For me I am okay with the way it is.

Reply:
mm..okay so enum is a type of DTO. then I am good @andrewsignori-aot

Reply:
IMO, We can skip the In/Out here i.e. RequestAssessmentTypeAPIDTO because in/out may not be different for this.

Reply:
IMO too, I am okay with the existing way and skip In/Out